File: 01-harassing-interpreter/main.go
package main

import "time"

func main(){
	total := 0
	start := time.Now()
	for i := 0; i < 1000000000; i++ {
		total += i
	}
	elapsed := time.Since(start)
	println(total)
	println(elapsed.Milliseconds())
}

File: 01-harassing-interpreter/main.py
import time


def main():
    total = 0
    start = time.time()
    for i in range(1000000000):
        total += i
    elapsed = time.time() - start
    print(total)
    print(elapsed*1000) # python time is in seconds, so we multiply by 1000 to get milliseconds

if __name__ == '__main__':
    main()